src/Polly.Extensions/DependencyInjection/AddResiliencePipelinesContext.cs

```diff
@@ -0,0 +1,73 @@
+﻿namespace Polly.DependencyInjection;
+
+/// <summary>
+/// Represents the context for configuring resilience pipelines with the specified key.
+/// </summary>
+/// <typeparam name="TKey">The type of the key used to identify the resilience pipeline.</typeparam>
+public class AddResiliencePipelinesContext<TKey>
```

Comment:
We can make this sealed given that there's no way for a public consumer to create a derived class as they can't access the constructor.

```suggestion
public sealed class AddResiliencePipelinesContext<TKey>
```